Demo/Demo/Networking/DemoMerchantAPI.swift

```diff
@@ -1,33 +1,29 @@
 import Foundation
+import PaymentsCore
 
 /// API Client used to create and process orders on sample merchant server
 final class DemoMerchantAPI {
 
     static let sharedService = DemoMerchantAPI()
-
+    var accessToken: String?
     private init() {}
-
```

Comment:
I think we still need this space?

Reply:
Seems like a lot of spacing in this file was deleted that we should revert so there is some seperation between items 😄

Reply:
spacings in in xcode is so confusing.. i think it's result of ctrl + i